### DISCUSSION TITLE
Release cadence for `poetry` and `poetry-core`?

### DISCUSSION THREAD

**Discussion by cnuss** [python-poetry/poetry#7068] [Q&A] [answered]
Hi there!

I'm eagerly awaiting some of these changes:
https://github.com/python-poetry/poetry-core/compare/1.3.2...main

What's the release cadence of `poetry` and `poetry-core`?

Cheers,
Christian

**Comment by cnuss** [comment]
cc @neersighted (I saw you answered a question in a prior discussion and are a owner of the Poetry organization :) ) for visibility on this question i have, thank you!

**Comment by neersighted** [comment, marked as answer]
As we can / when we can. The thing to keep in mind with core releases is Poetry is pinning to the patch level, so you'll only benefit from a new release without a Poetry release if you're consuming it through a non-Poetry frontend.

We may revisit that decision in the future: currently core has a simplified release structure where we generally try to support the stable and development branches with the same core branch. We can easily put infrastructure in place to support multiple core branches, and may do so -- it just has been significantly easier for the maintainers to not do so, so far.

https://github.com/python-poetry/poetry/issues/6972 is tracking the Poetry 1.3 release -- we plan a core release to be consumed by Poetry 1.3/support that release, with a schedule of "as soon as maintainer prepares it."

**Reply by cnuss** [reply]
@neersighted That's fine! A `poetry-core` release is all I really need, all of our `build-system.requires` in `pyproject.toml`'s reference `poetry-core` (not `poetry`). What would the ETA for a `poetry-core` release be?

**Reply by neersighted** [reply]
> with a schedule of "as soon as maintainer prepares it."

**Reply by cnuss** [reply]
@neersighted i'm trying to get some clarity for timing and this project's scheduling. 

so when will the next "maintainer prepare it?" for `poetry-core`? is it based on the checklist in #6972, or since the `poetry-core` version is locked and all CI checks are green, it could be today or in the next few days? or weeks?

**Reply by neersighted** [reply]
Apologies, I thought it could be inferred from the linked conversation. To be painfully explicit:

Core is ready for a 1.4 release. Poetry is not ready for a 1.4 core (or rather, will need some work to work with it), but the first step will be releasing it, and then merging some minor PRs (slash unblocking some PRs otherwise mergable) in Poetry. 

Core can be released any time, from the next hour, to the next 6 months, based on maintainer energy and availability.

**Reply by cnuss** [reply]
Thanks for the clarity! 😃 

Is there a maintainer I can reach out to for a 1.4 release of `poetry-core` to gauge their energy and availability?

**Reply by neersighted** [reply]
Myself, @radoering, @finswimmer, and @abn are the usual suspects for preparing core releases. My availability is limited this week due to the US holidays.

@Secrus had expressed interest in preparing the release in our last maintainer's call, but is also very busy at work as I understand it.

I would refrain from pinging any of the named subjects again -- this comment ought to be more than sufficient to both express interest and provide a nudge.

**Reply by neersighted** [reply]
https://github.com/python-poetry/poetry-core/releases/tag/1.4.0

**Reply by cnuss** [reply]
thank you!!
